maps/away/liberia/liberia.dm

```diff
@@ -25,7 +25,7 @@
 	vessel_mass = 3000
 	fore_dir = WEST
 	max_speed = 1/(1 SECOND)
-	known = 1
+	sector_flags = OVERMAP_SECTOR_KNOWN
```

Comment:
pretty useless to set mobile ship ot known since it'll only gives its spawn position to helm consoles

Reply:
I think that's the idea, so they can head towards it? Not too sure of the logic, just wanted to make it 1:1.